target/linux/apm821xx/dts/meraki-mr24.dts

```diff
@@ -110,6 +127,9 @@
 &EMAC0 {
 	status = "okay";
 
+	nvmem-cells = <&macaddr_factory_66>;
```

Comment:
you don't need to bother with the MR24. u-boot will fill out the mac-address property with the correct value.

(u-boot will look up the ethernet0 alias in the chosen node and update/add/replace whatever is in the mac-address property value)

Reply:
And nope. that was the MBL. Different device. Sorry!